.env.mainnet

```diff
@@ -20,7 +20,7 @@ HASURA_GRAPHQL_ACTION_BASE_URL=http://hapi:9090
 
 # hapi
 HAPI_EOS_API_NETWORK_NAME=mainnet
-HAPI_EOS_API_ENDPOINT=https://eos.edenia.cloud
+HAPI_EOS_API_ENDPOINTS=["https://eos.edenia.cloud","https://eos.api.eosnation.io","https://eos.greymass.com","https://eos.eosphere.io","https://api.eosrio.io"]
```

Comment:
i find the aloha EOS endpoint to be very good and dosen't rate limit as much. can we add it to the list: https://api.main.alohaeos.com